Docker/Jenkins-Worker/Dockerfile

```diff
@@ -34,6 +34,11 @@ RUN apt-get update \
      software-properties-common \
      sudo \
   && ln -s /usr/bin/lua5.3 /usr/local/bin/lua
+ 
+# install git 
+RUN apt-get update && \
```

Comment:
Looks like it didn't work:
```
6/1/2021, 6:02:14 PM git is already the newest version (1:2.20.1-2+deb10u3). 0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
6/1/2021, 6:02:23 PMRemoving intermediate container ca2b585020da
```
I wonder if this would be more effective:
```
apt-get -t=2.28.0 install git
```